prow/plugins/projectmanager/projectmanager_test.go

```diff
@@ -0,0 +1,257 @@
+						},
+						{
+							Name: "label2",
+						},
+					},
+				},
+				// Note that repoProjects and ProjectColumns are empty so the columnID cannot be looked up using repo, project and column names
+				// This means if the project_manager plugin is not using the columnID specified in the config this test will fail
+				RepoProjects:   map[string][]github.Project{},
+				OrgProjects:    map[string][]github.Project{},
+				ProjectColumns: map[int][]github.ProjectColumn{},
+				ColumnCardsMap: map[int][]github.ProjectCard{},
+			},
+			projectManager: plugins.ProjectManager{
+				OrgRepos: map[string]plugins.ManagedOrgRepo{
+					"testOrg/testRepo": {
+						Projects: map[string]plugins.ManagedProject{
+							"testProject": {
+								Columns: []plugins.ManagedColumn{
+									{
+										ID:     columnID,
+										Name:   "testColumn",
+										State:  "open",
+										Org:    "otherOrg",
+										Labels: []string{"label1", "label2"},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			pe: github.PullRequestEvent{
+				Action: github.PullRequestActionOpened,
+				Number: 1,
+				PullRequest: github.PullRequest{
+					ID:    2,
+					State: "open",
+				},
+				Repo: github.Repo{
+					Name: "someRepo",
+					Owner: github.User{
+						Login: "otherOrg",
+					},
+				},
+			},
+			expectedColumnCards: map[int][]github.ProjectCard{
+				1: {
+					{
+						ContentID:   2,
+						ContentType: "PullRequest",
+					},
+				},
+			},
+		},
+		{
+			name: "don't add pull request with incorrect labels",
+			gc: &fakegithub.FakeClient{
+				OrgRepoIssueLabels: map[string][]github.Label{
+					"otherOrg/someRepo/1": {
+						{
+							Name: "label1",
+						},
+					},
+				},
+				RepoProjects: map[string][]github.Project{
+					"testOrg/testRepo": {
+						{
+							Name: "testProject",
+							ID:   1,
+						},
+					},
+				},
+				OrgProjects: map[string][]github.Project{},
+				ProjectColumns: map[int][]github.ProjectColumn{
+					1: {
+						{
+							Name: "testColumn",
+							ID:   1,
+						},
+					},
+				},
+				ColumnCardsMap: map[int][]github.ProjectCard{},
+			},
+			projectManager: plugins.ProjectManager{
+				OrgRepos: map[string]plugins.ManagedOrgRepo{
+					"testOrg/testRepo": {
+						Projects: map[string]plugins.ManagedProject{
+							"testProject": {
+								Columns: []plugins.ManagedColumn{
+									{
+										Name:   "testColumn",
+										State:  "open",
+										Org:    "otherOrg",
+										Labels: []string{"label1", "label2"},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			pe: github.PullRequestEvent{
+				Action: github.PullRequestActionOpened,
+				Number: 1,
+				PullRequest: github.PullRequest{
+					ID:    2,
+					State: "open",
+				},
+				Repo: github.Repo{
+					Name: "someRepo",
+					Owner: github.User{
+						Login: "otherOrg",
+					},
+				},
+			},
+			expectedColumnCards: map[int][]github.ProjectCard{},
+		},
+	}
```

Comment:
Please add test cases for
- config for multiple repos in the same org (show that only the relevant config applies)
- config at both the org and repo level (show that both levels apply)
- a PR that should add multiple column cards

Reply:
Done